src/Core/src/Platform/iOS/MauiImageView.cs

```diff
@@ -7,18 +7,34 @@ namespace Microsoft.Maui.Platform
 {
 	public class MauiImageView : UIImageView
 	{
+		WeakReference<ImageHandler>? _handler;
+
+		public MauiImageView(ImageHandler handler) => _handler = new WeakReference<ImageHandler>(handler);
+
+		[Obsolete("To be removed in a future release")]
 		public MauiImageView()
 		{
 		}
 
+		[Obsolete("To be removed in a future release")]
 		public MauiImageView(CGRect frame)
 			: base(frame)
 		{
 		}
 
-		public override void MovedToWindow() =>
-			WindowChanged?.Invoke(this, EventArgs.Empty);
+		public override void MovedToWindow()
+		{
+			if (_handler is not null && _handler.TryGetTarget(out var handler))
+			{
+				handler.NotifyWindowChanged();
```

Comment:
I am not 100% happy with views reaching out into internal handlers as this means this view will never work for anything else, but for now maybe it is fine? The method of catching the iOS window change is not great and is a bit iffy - we could actually do all this in cross platform code. We now have a Window property. But this is more for another issue.

@PureWeen thoughts?

Reply:
If we made this one public on `IImageHandler` and gave it a proper name, it would mean other `IImageHandler` could work?